traffic_portal/app/src/common/modules/table/certExpirations/TableCertExpirationsController.js

```diff
@@ -17,27 +17,24 @@
  * under the License.
  */
 
-var TableCertExpirationsController = function(tableName, certExpirations, dsXmlToIdMap, filter, $document, $scope, $state, $filter, locationUtils, certExpirationsService) {
-
-	let table;
+var TableCertExpirationsController = function(tableName, certExpirations, dsXmlToIdMap, filter, $document, $scope, $state, $filter, locationUtils) {
+
+	/** All of the expiration fields converted to actual Dates */
+	$scope.certExpirations = certExpirations.map(
+		function(x) {
+			// need to convert this to a date object for ag-grid filter to work properly
+			x.expiration = new Date(x.expiration.replace("+00", "Z"));
```

Comment:
The timestamps returned by the API are RFC3339 with nanosecond precision, so this replacement is unnecessary.